src/Components/missions/Missions.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { toggleJoining } from '../../redux/Missions/missionsReducer';
+import style from './missions.module.css';
+
+function Missions() {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <table className={style.table} data-testid="list-of-missions">
+        <thead className={style.thead}>
+          <tr className={style.header}>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        {missions.map((el) => (
+          <tbody key={el.id} className={style.item}>
```

Comment:
- [ ] As the previous reviewer mentioned kindly consider making the UI look precisely like the provided wireframe this will make you a better programmer by reproducing the client design

|Your app|Provided design|
|-|-|
|![Screenshot_20221119_085417](https://user-images.githubusercontent.com/25227022/202840913-7a639548-f2d3-4280-a905-6b857d9cc271.png)|![Screenshot_20221119_084436](https://user-images.githubusercontent.com/25227022/202840876-350379da-e99f-434c-a0d8-4c2bedb41c75.png)|